src/_pytest/pathlib.py

```diff
@@ -334,3 +335,12 @@ def fnmatch_ex(pattern, path):
 def parts(s):
     parts = s.split(sep)
     return {sep.join(parts[: i + 1]) or sep for i in range(len(parts))}
+
+
+def unique_path(path):
```

Comment:
Was it considered to keep this private, i.e. as `_unique_path`?

Reply:
It was originally private, but given it was on a private module already, I asked @Oberon00 to remove the `_` prefix for consistency with other functions in the module.

Reply:
See https://github.com/pytest-dev/pytest/pull/5792/commits/29bb0eda276fc3808264a1b070544339c1eee393